From 84e39c3615650a5898d098369fce56c52fe0a5c6 Mon Sep 17 00:00:00 2001
From: Mohammed Hassan <mohammed.hassan@jolla.com>
Date: Tue, 28 Jul 2015 15:38:16 +0300
Subject: [PATCH 1/3] Mer: disable camerabin previews handling

Our camera source does not yet support generating previews thus enabling it without
a proper support from the source leads to issues with processing counter and camerabin
never reaching idle state.
---
 src/plugins/gstreamer/camerabin/camerabinsession.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/gstreamer/camerabin/camerabinsession.cpp b/src/plugins/gstreamer/camerabin/camerabinsession.cpp
index 0402eee2..d70663d9 100644
--- a/src/plugins/gstreamer/camerabin/camerabinsession.cpp
+++ b/src/plugins/gstreamer/camerabin/camerabinsession.cpp
@@ -181,7 +181,7 @@ CameraBinSession::CameraBinSession(GstElementFactory *sourceFactory, QObject *pa
         g_object_set(G_OBJECT(m_camerabin), "flags", envFlags.toInt(), NULL);
 
     //post image preview in RGB format
-    g_object_set(G_OBJECT(m_camerabin), POST_PREVIEWS_PROPERTY, TRUE, NULL);
+    g_object_set(G_OBJECT(m_camerabin), POST_PREVIEWS_PROPERTY, FALSE, NULL);
 
 #if GST_CHECK_VERSION(1,0,0)
     GstCaps *previewCaps = gst_caps_new_simple(
-- 
2.15.1

